pkg/util/env/env.go

```diff
@@ -21,12 +21,12 @@ import (
 	"k8s.io/klog/v2"
 )
 
-// NodeNameEnvKey is environment variable.
 const (
-	NodeNameEnvKey     = "NODE_NAME"
-	podNameEnvKey      = "POD_NAME"
-	podNamespaceEnvKey = "POD_NAMESPACE"
-	svcAcctNameEnvKey  = "SERVICEACCOUNT_NAME"
+	NodeNameEnvKey        = "NODE_NAME"
+	podNameEnvKey         = "POD_NAME"
+	podNamespaceEnvKey    = "POD_NAMESPACE"
+	svcAcctNameEnvKey     = "SERVICEACCOUNT_NAME"
+	antreaConfigMapEnvKey = "ANTREA_CONFIG_MAP_NAME"
```

Comment:
The other variables don't use `antreaSvcAcctNameEnvKey` and `ANTREA_SERVICEACCOUNT_NAME`, maybe keep same here? i.e.:
```
configMapEnvKey = "CONFIGMAP_NAME"
```

Reply:
I am following the name used in downstream [here](https://gitlab.eng.vmware.com/core-build/mirrors_github_antrea/-/commit/8af39c44309b668809ef66d00e122038f5f2e954)  which is provided by @antoninbas , I guess we should keep them the same for less maintenance effort?

Reply:
I see. No strong opinion.